orchagent/notifications.cpp

```diff
@@ -23,12 +23,12 @@ void on_bfd_session_state_change(uint32_t count, sai_bfd_session_state_notificat
     // which causes concurrency access to the DB
 }
 
-void on_switch_shutdown_request()
+void on_switch_shutdown_request(sai_object_id_t switch_id)
```

Comment:
According to sairedis code, this method must have a switch_id parameter:

        switchNotifications.on_switch_shutdown_request(m_switchId);